### DISCUSSION TITLE
[Question] read tags from AWS EC2

### DISCUSSION THREAD

**Discussion by AchimGrolimund** [signalfx/splunk-otel-collector#2121] [General]
Hey all

how can i read all TAGs from an AWS-EC2 server? Only with an installed Splunk-otel-collector?

Do i need to enable some stuf on the AWS part or is it a MUST to enable the Dataintegration for the AWS-Account to SignalFx cloud platform

Greetings
Achim

**Comment by atoulme** [comment]
You can read about AWS EC2 tag collection here: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/README.md#aws-ec2
